Area Calculator Osu.py

```diff
@@ -0,0 +1,121 @@
+import time
+import array
+import pyautogui
+from statistics import median
+
+SAMPLE_RATE = 0.01
+GRACE_PERIOD = 5.0
+
+def calculate_size(current_area_size: float, normalized_playfield_size: float, inputs: array) -> float:
+
+    playfield_inputs = array.array('f')
+
+    # Remove everything outside the playfield
+    for input in inputs:
+        if input < normalized_playfield_size * 1.03125:
+            playfield_inputs.append(input)
+
+    max_input = max(playfield_inputs)
+    min_input = min(playfield_inputs)
+
+    inputs9 = array.array('f')
+    inputs1 = array.array('f')
+
+    for input in playfield_inputs:
+        if input > max_input * 0.9:
+            inputs9.append(input)
+
+        if input < min_input * 0.9:
+            inputs1.append(input * -1)
+
+    del max_input, min_input, playfield_inputs
+
+    median_9: float = median(inputs9)
+    median_1: float = median(inputs1)
+
+    played_area = median_9 + median_1
+
+    if played_area - (normalized_playfield_size * 2) < 0.1 and played_area - (normalized_playfield_size * 2) > -0.1:
+        return current_area_size
+    
+    corrected_area: float = current_area_size * played_area / (normalized_playfield_size * 2)
+    
+    return corrected_area
+
+
+def main():
+    is_running: bool = True
+
+    normalized_inputs_x = array.array('f')
+    normalized_inputs_y = array.array('f')
+
+    screen_size_x, screen_size_y = pyautogui.size()
+
+    playfield_size_y = screen_size_y * 0.8
+    playfield_size_x = playfield_size_y / 3 * 4
+
+    print("\nRemember to remove all the filters\n")
+    time.sleep(GRACE_PERIOD)
+
+    tablet_size_x = float(input("Enter the width of the tablet in mm (. for decimals): "))
```

Comment:
No input validation, this will throw with a random input (also, I could easily put `-1.0` and it would be ok). 
Please keep typer for that purposes, it's much easier. This can be applied to every single input in the source code.